interpreter/cling/tools/plugins/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+#------------------------------------------------------------------------------
+# CLING - the C++ LLVM-based InterpreterG :)
+#
+# This file is dual-licensed: you can choose to license it under the University
+# of Illinois Open Source License or the GNU Lesser General Public License. See
+# LICENSE.TXT for details.
+#------------------------------------------------------------------------------
+
+## If we drop a compatible cmake project in this folder we should automatically
+## pick it up and build it.
+#function(LISTSUBDIRS result curdir)
+#  file(GLOB children RELATIVE ${curdir} ${curdir}/*)
+#  set(dirlist "")
+#  foreach(child ${children})
+#    if(IS_DIRECTORY ${curdir}/${child})
+#      if(APPEND dirlist ${child})
+#    endif()
+#  endforeach()
+#  set(${result} ${dirlist} PARENT_SCOPE)
+#endfunction()
+#
+#LISTSUBDIRS(subdirs ${CMAKE_CURRENT_SOURCE_DIR}/plugins/)
+#foreach(subdir ${SUBDIRS})
+#  add_subdirectory(${subdir})
+#endforeach()
+
+ExternalProject_Add(
+  clad
+  GIT_REPOSITORY https://github.com/vgvassilev/clad.git
+  GIT_TAG master
```

Comment:
I don't think this can fly. ROOT 6.14.00 compiled in 2020 will still check out the master of clad, and that will fail to build against 2018's cling, llvm and clang. Maybe use a commit sha1 here?

Reply:
Sure, that's not there to stay for eternity. This is more like work in progress. I am planning to tag once we have a tiny demonstrator (say in ROOT math).